pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -163,15 +163,13 @@ def cmd(self):
             'pytest',
             '--ds={}'.format('{}.envs.{}'.format(self.root, self.settings)),
             "--junitxml={}".format(self.xunit_report),
+            '-vvv',
+            '-s',
+            '--create-db',
+            '--migrations',
         ])
         cmd.extend(self.test_options_flags)
-        if self.verbosity < 1:
-            cmd.append("--quiet")
-        elif self.verbosity > 1:
-            cmd.append("--verbose")
 
-        if self.disable_capture:
-            cmd.append("-s")
```

Comment:
This name confused me.  `-s`, which I added elsewhere, was needed to see the migration output.

Reply:
Note: I still think `disable_capture` is a confusing name, but I don't plan to change it.  I did reference it from the update in the doc file.